src/main/resources/com/typesafe/sbt/packager/archetypes/systemloader/systemd/start-template

```diff
@@ -6,7 +6,7 @@ Requires=${{start_facilities}}
 Type=simple
 WorkingDirectory=${{chdir}}
 EnvironmentFile=${{env_config}}
-ExecStart=${{chdir}}/bin/${{exec}}
+ExecStart=${{chdir}}/bin/${{exec}} $RUN_OPTS
```

Comment:
This should already work. The `EnvironmentFile` above provides the ability to add environment variables.

In systemv this always felt like a hack.